angular/projects/spark-angular/src/lib/directives/sprk-box/sprk-box.stories.ts

```diff
@@ -0,0 +1,46 @@
+import { storyWrapper } from '../../../../../../.storybook/helpers/storyWrapper';
+import { SprkBoxDirective } from './sprk-box.directive';
+import { SprkBoxModule } from './sprk-box.module';
+import { markdownDocumentationLinkBuilder } from '../../../../../../../storybook-utilities/markdownDocumentationLinkBuilder';
+
+export default {
+  title: "Components/Box",
+  component: SprkBoxDirective,
+  decorators: [storyWrapper(storyContent => `<div class="sprk-o-Box sb-decorate">${storyContent}<div>`)],
+  parameters: {
+    info: `
+${markdownDocumentationLinkBuilder('box')}
+Box is a layout component that separates a group of content from its surroundings.
+
+Use Box when you have a group of content that needs to be visually separated from other content on the page through padding.
+
+Since the effects of Box are only seen through whitespace, the examples have a background color to illustrate how Box works.
+
+Box has built in padding sizes that are paired with our Spacing values. Otherwise, it will default to medium spacing.
```

Comment:
Could we add a link to the Spacing page here?

Reply:
the class modifiers section has the spacing value, and it feel that's more relevant. Linked in the sentence after it.